src/locale/locales/fr/messages.po

```diff
@@ -865,6 +869,10 @@ msgstr "Flouter les images et les filtrer des fils d’actu"
 msgid "Books"
 msgstr "Livres"
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:614
+msgid "Brag a little!"
+msgstr "Vantez un peu !"
```

Comment:
Yikes, that would work in the 1930s I think but not today. The most equivalent would be `Crânez un peu !` but it's too familiar, or `Fanfaronnez-donc !` but it would be WAY too formal (it would work if you said that in a French embassy, maybe?).

Let's workaround by saying basically the same thing but using another structure, yet again.

```suggestion
msgstr "Soyez-en fier !"
```

Reply:
SGTM